README.md

```diff
@@ -1,6 +1,8 @@
 # K-9 Mail
-[![Build Status](https://k9mail.ci.cloudbees.com/job/master/badge/icon)](https://k9mail.ci.cloudbees.com/job/master/)
-[![Join the chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/k9mail/k-9)
+[![Build status](https://img.shields.io/travis/k9mail/k-9.svg?style=flat-square)](https://travis-ci.org/k9mail/k-9)
+[![Latest release](https://img.shields.io/github/release/k9mail/k-9.svg?style=flat-square)](https://github.com/k9mail/k-9/releases/latest)
+[![Join the chat](https://img.shields.io/gitter/room/k9mail/k-9.svg?style=flat-square)](https://gitter.im/k9mail/k-9)
+[![License](https://img.shields.io/github/license/k9mail/k-9.svg?style=flat-square)](https://raw.githubusercontent.com/k9mail/k-9/master/LICENSE)
```

Comment:
GitHub already displays the license type. We don't need a separate badge for that.